src/alchemlyb/convergence/convergence.py

```diff
@@ -122,23 +117,11 @@ def forward_backward_convergence(df_list, estimator="MBAR", num=10, **kwargs):
         sample = []
         for data in df_list:
             sample.append(data[-len(data) // num * i :])
-        sample = concat(sample)
-        result = my_estimator.fit(sample)
-        if estimator == "MBAR":
-            my_estimator.initial_f_k = result.delta_f_.iloc[0, :]
```

Comment:
by deleting this block, aren't you undoing some of the changes from PR #357, namely the CHANGES entry
> forward_backward_convergence` uses the result from the previous step as the initial guess for the next step. (PR #357)

Please confirm that this is what you want to do, and if so, update CHANGES.